blackbox/docs/sql/statements/set.rst

```diff
@@ -47,6 +49,20 @@ logging level.
 ``SET SESSION/LOCAL`` are introduced to be compliant with third-party
 applications which use the PostgresSQL wire protocol.
 
+``SET LICENSE`` registers :ref:`License specific Settings <conf-license>`
+for license verification. If the verification succeeds, the cluster will
+continue to operate in `Enterprise Edition`_ until the license expiry date.
+
+.. NOTE::
+
+    Should the license verification fails, the cluster will continue to
+    operate in the existing mode. e.g. ``Enterprise Trial`` if the current license
```

Comment:
drop "." after mode as that separates the context of the "next sentence" that is not a sentence on its own